my-app/src/components/header/header.tsx

```diff
@@ -1,21 +1,25 @@
 import React from 'react';
-import './header.css'
+import { makeStyles } from '@mui/styles';
 
-const styles = {
-    title: {
-      color: "white",
-      fontWeight: "bold",
-      fontSize: "2rem",
-    },
-  };
+const useStyles = makeStyles({
+  title: {
+    color: "white",
+    fontWeight: "bold",
+    fontSize: "2rem",
+  },
+});
 
-function Header(){
-    return (
+function Header() {
```

Comment:
Top tip about exports in components - don't use `default` exports to make it easier to rename components in the future.
The problem with using `default` exports is because the import of such component looks like this:
```
import Header from './components/header';
```
Once you rename the component the IDE will normally keep the import name:
```
import Header from './components/app-header';
```
Instead please use named exports, which will force the IDE to also rename your named import to:
```
import { AppHeader } from './components/app-header';
```